Gemfile

```diff
@@ -93,3 +93,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: %w[mingw mswin x64_mingw jruby]
+
```

Comment:
can you move this gem declaration (and the one for tzinfo) to one of the sections near the top of the file?

Reply:
Done!